src/methods/geom_relations/geom_geom_processors.jl

```diff
@@ -146,7 +137,7 @@ function _line_curve_process(
         for j in (closed_curve ? 1 : 2):nc
             c_end = _tuple_point(GI.getpoint(curve, j))
             # Check if line and curve segments meet
-            seg_val = _segment_segment_orientation((l_start, l_end), (c_start, c_end))
+            seg_val, intr1, _ = _intersection_point(Float64, (l_start, l_end), (c_start, c_end))
```

Comment:
For the future, maybe we could make this something like 
```julia
seg_val, intr1, _ = _intersection_point([ExactMath()/FastMath()], ...)
```
so that a math type indicator can be passed down smoothly.